src/components/Accordion/src/Accordion.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+	<div
```

Comment:
Should consider all the recommendations from W3C on the markup, attributes and controls. https://www.w3.org/WAI/ARIA/apg/patterns/accordion/